### ISSUE TITLE
Attributes feature.

### ISSUE THREAD

**Issue by armcburney** [emerald-lang/emerald#16]
Need some advice for this one.

I had originally thought of doing attributes something like this:

```
html
  head
    title Attributes Example

  body
    h1 Attributes follow parentheses. (
      id     'header'
      class  'main-text'
      height '50px'
      width  '200px'
    )

  footer
    p Copyright Emerald Language 2016
```

I like the idea of having each attribute on a separate line, to try to mitigate long lines.  I had figured that would look cleaner than:

```
html
  head
    title Attributes Example

  body
    h1(id 'header', class 'main-text', height '50px', width '200px') Some text after attributes

  footer
    p Copyright Emerald Language 2016
```

But I'm not sure, for longer lines, I don't know if the first way would look visually appealing.

I still like the idea of having each attribute on its own line, and having iteroperability with styles, and events - like this early emerald code:

```
    main
      section
        h1 Why use Emerald?
        p -> Emerald allows you to scope events and styles to html
          elements in an elegant, clean way.

      figure
        figcaption -> Here's an example of elements scoped in a
          button here.

        button Click me. (
          click -> console.log("I was clicked!")
          hover -> console.log("I was hovered!")
        )

```

What are your thoughts @davepagurek, @icechen1?

**Comment by davepagurek** [comment]
Not sure this is _good_, but it's an option:

```
button (
  click -> console.log("I was clicked!")
  hover -> console.log("I was hovered!")
) ->
  Click me.
```

It's kind of gross though, hmm

**Comment by davepagurek** [comment]
additional idea, we could use the above syntax (tag, attributes, multiline literal) if it gets too long and allow using the original syntax (tag, content, attributes) for short lines

**Comment by armcburney** [comment]
Yeah, I like that idea @davepagurek.

**Comment by armcburney** [comment]
This issue was addressed by #28.
